migrations/sql/V2020.06.08.13.30__add_updated_date_to_now_tables.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE now_application ADD COLUMN last_updated_by character varying(60);
+ALTER TABLE now_application ADD COLUMN IF NOT EXISTS last_updated_date timestamp with time zone DEFAULT NOW() NOT NULL;
```

Comment:
I don't think there should be a default value for this column and it should be nullable.

Reply:
I'll remove the default! I agree